app/assets/javascripts/discourse/tests/acceptance/category-edit-test.js

```diff
@@ -113,10 +113,12 @@ acceptance("Category Edit", function (needs) {
     const allowedTagChooser = selectKit("#category-allowed-tags");
     await allowedTagChooser.expand();
     await allowedTagChooser.selectRowByValue("monkey");
+    await allowedTagChooser.collapse();
 
     const allowedTagGroupChooser = selectKit("#category-allowed-tag-groups");
     await allowedTagGroupChooser.expand();
     await allowedTagGroupChooser.selectRowByValue("TagGroup1");
+    await allowedTagGroupChooser.collapse();
```

Comment:
These new `.collapse()` calls are needed because the tests now fail when calling `.expand()` repeatedly on the same element.